server/src/main.rs

```diff
@@ -46,6 +51,18 @@ async fn main() -> Result<()> {
             .data(graphql::make_schema(settings.clone(), pool.clone()))
             .wrap(middleware::Compress::default())
             .wrap(middleware::Logger::default())
+            .wrap(IdentityService::new(
+                // <- create identity middleware
+                CookieIdentityPolicy::new(&[0; 32]) // <- create cookie identity policy
+                    .name("stacksexchange-auth-cookie")
+                    .secure(false),
+            )) // TODO(change to true one shit is working)
```

Comment:
Let's be specific about which shit needs to be working lol

Reply:
I realized this doesn't need to be a part of this PR. removed.